File: Python Programming/prime.py
import math
store=[]

a=int(input("Enter the lower limit: "))
b=int(input("Enter the upper limit: "))
print("The prime numbers in the range are: ")
for number in range(a,b+1):
    if number > 1:
        for i in range(2,number):
            if(number%i)==0:
                break
        else:
            print(number)
            store.append(number)
        
print("The number of prime numbers are: ",len(store))

File: Java Programming/prime.java
import java.util.Scanner;

public class prime{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int n, i;
        boolean flag = true;
        System.out.print("Enter no: ");
        n = input.nextInt();
        for(i=2;i<n;i++){
            if(n%i==0){
                flag = false;
                break;
            }
        }
        if(flag == true){
            System.out.println(n+" is a prime number");
        }
        else{
            System.out.println(n+"is not a prime number");
        }
    }
}